src/test/java/stashpullrequestbuilder/stashpullrequestbuilder/StashCauseTest.java

```diff
@@ -0,0 +1,119 @@
+package stashpullrequestbuilder.stashpullrequestbuilder;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.object.HasToString.hasToString;
+import static org.junit.Assert.assertThat;
+
+import org.junit.Test;
+
+public class StashCauseTest {
+
+  @Test
+  public void short_description_doesnt_allow_injection() {
+
+    StashCause stashCause =
+        new StashCause(
+            "",
+            "",
+            "",
+            "",
+            "",
+            PULL_REQUEST_ID,
+            OWNER,
+            DESTINATION_REPOSITORY_NAME,
+            PULL_REQUEST_TITLE,
+            "",
+            "",
+            "",
+            "",
+            null);
+
+    assertThat(
+        stashCause.getShortDescription(),
+        is(
+            "<a href='&#x2f;projects&#x2f;owner&#x25;3C&#x25;3E&amp;&#x27;&#x25;22&#x2f;repos&#x2f;name&#x25;3C&#x25;3E&amp;&#x27;&#x25;22&#x2f;pull-requests&#x2f;id&#x25;3C&#x25;3E&amp;&#x27;&#x25;22'>"
+                + "PR &#x23;id&lt;&gt;&amp;&#x27;&quot; title&lt;&gt;&amp;&#x27;&quot; </a>"));
```

Comment:
A shorter string would be more readable. We are testing that the escaping is done at all. It would be too much to test the underlying quoting algorithm.